kinto/__main__.py

```diff
@@ -98,16 +118,18 @@ def main(args=None):
                 import pip
                 pip.main(['install', "kinto[postgresql]"])
 
-    elif parsed_args['which'] == 'migrate':
+    elif which_command == 'migrate':
         dry_run = parsed_args['dry_run']
-        try:
-            logging.config.fileConfig(config_file)
-        except configparser.NoSectionError as e:  # pragma: no cover
-            print(e)
```

Comment:
I think we should add [verbose options](https://github.com/Kinto/kinto-http.py/blob/master/kinto_http/cli_utils.py#L65-L76) then, but the logger config of the server  with the HekaLogger doesn't fit well with running this command.